openstack/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG - openstack
 
+## 1.6.0 / 2018-10-12
+
+* [Added] Adds ability to Trace "check" function with DD APM. See [#2079](https://github.com/DataDog/integrations-core/pull/2079).
```

Comment:
This should be removed and we shouldn't release this check. This tracing was reverted in a later PR and is no longer on master.